guide/14-deep-learning/FCN_MLModel_guide-final_draft.ipynb

```diff
@@ -0,0 +1,501 @@
+{
```

Comment:
<p>(1) &nbsp;it automatically does this <s>in</s>  <strong>for the</strong> variable predict</p><p>(2) by the user in case of explanatory <strong>variables,</strong> <s>,, </s>however in the variable predict we (Note: remove extra comma)</p><p>(3) considered as a classification<strong> case, otherwise, </strong>it treats it as a regression case. <strong>In addition,</strong> this also provides a check for data types of fields.&nbsp;</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/718/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>